guides/typography.md

```diff
@@ -51,8 +51,8 @@ table below.
 | `headline-6`     | Section heading corresponding to the `<h2>` tag.                            |
 | `subtitle-1`     | Section heading corresponding to the `<h3>` tag.                            |
 | `subtitle-2`     | Section heading corresponding to the `<h4>` tag.                            |
-| `body-1`         | Base body text.                                                             |
-| `body-2`         | Bolder body text.                                                           |
+| `body-1`         | Bolder body text.                                                             |
```

Comment:
You're right that the descriptions here are out of date, they're actually the same boldness now. I think "Base body text" is correct for body-1, for body-2 I would change it to something like "Smaller secondary body text"